### PR TITLE
Add: ROADMAP.md

### PR DESCRIPTION
**Pull request by mlavi**
## Change Overview

Adding ROADMAP.md from our draft discussion.

## Pull request type

Please check the type of change your PR introduces:

- [x] :hamster: Trivial/Minor

## Test Plan

- [X] :muscle: Manual

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Project and [Governance](Governance.md) Work
+
+1. Lifecycle for contributors: Roles, Privs, and lifecycle
+   1. project admin?
+   2. maintainer (core)
+   3. reviewer+approval: PR + branch protections?
+   4. contributor
+2. Blueprints Maintenance and Support Policy
+   1. project test matrix: Kopia vs Restic vs Stow, vs K10 vs Kanister
```
Comment (pavannd1) [inline ROADMAP.md:14]:
```suggestion
   1. project test matrix: Kopia vs Restic vs Stow, downstream adopters vs Kanister standalone
```

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   2. community maintained examples: move to a new public repo
+3. Move Kanister.io web site to GitHub Repo
```
Comment (pavannd1) [inline ROADMAP.md:16]:
```suggestion
3. Move the Kanister.io website to GitHub Repo
```

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+4. Leverage issues for planning?
```
Comment (pavannd1) [inline ROADMAP.md:17]:
```suggestion
4. Leverage GitHub Issues and Projects for planning
```

File: ROADMAP.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Development Work
+
+### Prioritized (Doing):
+
+1. Fork Block functions, deprecate unsed Kanister code
```
Comment (pavannd1) [inline ROADMAP.md:23]:
```suggestion
1. Fork block storage functions, deprecate unused Kanister code
```

File: ROADMAP.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+2. Kopia.io Repository Controller with a CR to control the lifecycle of a Kopia Repository Server
+3. [Progress tracking for individual Phases in an Action](https://github.com/kanisterio/kanister/blob/master/design/progress-tracking.md)
+4. ActionSet metrics
```
Comment (pavannd1) [inline ROADMAP.md:26]:
```suggestion
4. ActionSet metrics
5. Container image vulnerability scanning
```

File: ROADMAP.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+### New Features:
+
+1. Track and log events triggered by Blueprint Actions
+2. Image Vulnerability Scanning
+3. ARM support
+4. Use GitHub pages with Jekyll or Hugo for documentation
+5. Vault integration for Repository Server secrets
```
Comment (pavannd1) [inline ROADMAP.md:34]:
```suggestion
1. Track and log events triggered by Blueprint Actions
2. ARM support
3. Use GitHub pages with Jekyll or Hugo for documentation
4. Vault integration for Repository Server secrets
```

File: ROADMAP.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+### Backlog: (ideas and maintenace)
```
Comment (pavannd1) [inline ROADMAP.md:36]:
```suggestion
### Backlog: (ideas and maintenance)
```

File: ROADMAP.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Deprecate Restic
+2. Generate Kanister controller using KubeBuilder - current one is based on rook operator
+3. Merge Repository controller into Kanister controller after a.
```
Comment (pavannd1) [inline ROADMAP.md:40]:
```suggestion
3. Merge the Repository controller into the Kanister controller after 2.
```

**Comment by mlavi** [comment]
@pavannd1 thanks for the improvements, please review!

**Review by pavannd1 — APPROVED** [review]

**Pull request merged**